include/clink/feature/one_hot_encoder.h

```diff
@@ -38,22 +37,28 @@ class OneHotEncoderModel {
   OneHotEncoderModel(const OneHotEncoderModel &other) = delete;
   OneHotEncoderModel &operator=(const OneHotEncoderModel &) = delete;
 
+  // TODO: Add a transform method with generic signature in the Model class
   // Converts the provided data to a SparseVector.
-  llvm::Expected<SparseVector> transform(const int value,
-                                         const int columnIndex);
+  tfrt::AsyncValueRef<SparseVector> transform(const int &value,
```

Comment:
Unlike string, whose copy could incur high overhead, we in general use copy constructor for int/long/bool/... types directly for simplicity instead of using `&`.

Could you update the PR to follow this convention?